clients/fides-js/src/fides.ts

```diff
@@ -112,42 +115,44 @@ async function init(this: FidesGlobal, providedConfig?: FidesConfig) {
     ...config,
     options: { ...config.options, ...overrides.optionsOverrides },
   };
-  const cookie = {
+  this.cookie = {
     ...getInitialCookie(config),
     ...overrides.consentPrefsOverrides?.consent,
   };
 
   // Keep a copy of saved consent from the cookie, since we update the "cookie"
   // value during initialization based on overrides, experience, etc.
-  const savedConsent: NoticeConsent = {
-    ...cookie.consent,
+  this.saved_consent = {
+    ...this.cookie.consent,
   };
 
   const initialFides = getInitialFides({
     ...config,
-    cookie,
-    savedConsent,
+    cookie: this.cookie,
+    savedConsent: this.saved_consent,
     updateExperienceFromCookieConsent: updateExperienceFromCookieConsentNotices,
   });
   if (initialFides) {
     Object.assign(this, initialFides);
     updateWindowFides(this);
-    dispatchFidesEvent("FidesInitialized", cookie, config.options.debug);
+    dispatchFidesEvent("FidesInitialized", this.cookie, config.options.debug, {
+      shouldShowExperience: this.shouldShowExperience(),
+    });
   }
-  const experience = initialFides?.experience ?? config.experience;
+
   const updatedFides = await initialize({
     ...config,
-    cookie,
-    savedConsent,
-    experience,
+    fides: this,
```

Comment:
OK, I think this is where this is failing in more E2E testing. I added some logging just before this call to `initialize(...)` like this:

```
  console.log(
    "DEBUG: calling initialize() with config, this, overrides...",
    config,
    this,
    overrides
  );

  const updatedFides = await initialize({
    ...config,
    fides: this,
    renderOverlay,
    updateExperience,
    overrides,
  });
```

In my test that outputs this (sorry, copy-pasting from console is hard here and not super helpful):
![image](https://github.com/ethyca/fides/assets/1834295/419040fd-8e87-4992-82b6-0372ecb2532e)

The issue is:
* at this point in the code, `this.config.experience = { ...configured experience object... }` but `this.experience` is still undefined
* when we do `{ ...config, fides }` and pass that down, you previously were received `experience` as an argument to `initialize()`, but now it's trying to look at `fides.experience`
* but nothing initializes `this.experience` to the value from the config object, so it's ignored

Reply:
Right, previously we had this line `const experience = initialFides?.experience ?? config.experience;` that handled when experience was passed on the config, so I think we'll just need to account for this still.

Reply:
Having issues running cypress admin-ui locally due to the following err, so I'm having to run this through github to test:
```
admin-ui:cy:open: We detected that the Chromium Renderer process just crashed.
admin-ui:cy:open:
admin-ui:cy:open: This can happen for a number of different reasons.
admin-ui:cy:open:
admin-ui:cy:open: If you're running lots of tests on a memory intense application.
admin-ui:cy:open:   - Try increasing the CPU/memory on the machine you're running on.
admin-ui:cy:open:   - Try enabling experimentalMemoryManagement in your config file.
admin-ui:cy:open:   - Try lowering numTestsKeptInMemory in your config file during 'cypress open'.
```

Reply:
I've been experiencing the same in the Cypress GUI 
![CleanShot 2024-05-23 at 09 12 21@2x](https://github.com/ethyca/fides/assets/103820/4fa5cf52-a329-4f76-8d45-a1988986db58)